src/components/Details/Details.tsx

```diff
@@ -23,23 +24,22 @@ interface SelectedMarketProps {
   id: string;
 }
 
-export const Details: React.FC<SelectedMarketProps> = ({
-  selectedMarket,
-  id
+export const Details: React.FC<SelectedMarketProps> = ({ 
+  selectedMarket, 
+  id 
 }) => {
-  const openSeason = `Season: ${selectedMarket?.schedule[0].season}`;
-
-  const daysAndTimes = selectedMarket?.schedule.map((sched, index) => {
-    return (
-      <div key={index}>
-        <ul className='day-list'>
-          <li>
-            {sched.dayOfWeek} {sched.time}
-          </li>
-        </ul>
-      </div>
-    );
-  });
+  let [error, setError] = useState("");
+  let [ openSeason, setOpenSeason ] = useState("")
+  let [ daysAndTimes, setDaysAndTimes ] = useState("")
+ 
+  useEffect(() => {
+    if (!selectedMarket?.schedule.length) {
+      setError("Uh oh! Market not found! Please try again.")
+    } else {
+      setOpenSeason(`${selectedMarket?.schedule[0].season}`);
+      setDaysAndTimes(`${selectedMarket?.schedule[0].dayOfWeek} ${selectedMarket?.schedule[0].time}`);
+    }
+  }, [])
```

Comment:
Yeah, there were some strange errors/issues before Ash moved the season and days into state. I think this could be revisited now that there are more changes in util, but we can't set JSX in state, which is why it looks like this at the moment.